docs/diagrams/OfferModelClassDiagram.puml

```diff
@@ -6,7 +6,7 @@ skinparam classBackgroundColor MODEL_COLOR
 
 UniqueOfferList *-right-> "*" Offer
 
-Offer *--> GoodName
-Offer *--> Price
+Offer *--> "1" GoodName
+Offer *--> "1" Price
```

Comment:
There is no UniqueOfferList, is this hypothetical or to be implemented soon?

Reply:
i think i copied blindly from ab3